src/imitation/scripts/config/parallel.py

```diff
@@ -52,15 +49,57 @@ def debug_log_root():
 
 @parallel_ex.named_config
 def example_cartpole_rl():
-  """Example config that spins up 8 different training runs of cartpole."""
   inner_experiment_name = "expert_demos"
+  outer_experiment_name = "example-cartpole"
+  n_seeds = 2
   search_space = {
     "config_updates": {
-      "seed": tune.grid_search([0, 1]),
       "init_rl_kwargs": {
-        "learning_rate": tune.grid_search([3e-4, 2e-4]),
-        "nminibatches": tune.grid_search([16, 32]),
+        "learning_rate": tune.grid_search([3e-4 * x for x in (1/3, 1/2, 1, 2)]),
```

Comment:
This is quite a narrow range to search for LR over. Normally you'd search in a log-uniform space, so e.g. `1e-x for x in range(2,4)`. If you're confident this is close to the right learning rate then fine to do a granular search.

Reply:
Noted, changing all the learning rates to search over the`np.logspace(3e-6, 1e-1, num=3)`. Keeping the number of searches low because, isn't an actual experiment, just an example that I can use to check if things are working properly.